releasenotes/notes/add-experimental-windows-networkv2-check-b19ce067314b3c1d.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add a networkv2 check that is a port of the Windows Python network check to
+    Go. This version is disabled by default but can be enabled with ``use_networkv2_check``
+    in your configuration.
```

Comment:
The check is still named `network`, not `networkv2`, and it's not a new check, the PR just adds the Windows version
Not super sure how to best explain that...

We might need to patch the [previous release note](https://github.com/DataDog/datadog-agent/pull/33190/files#diff-c05dd63dc559c750c4230a183109a9af6d74fb97962b8fd17665e3e9f42f6022) to make it clearer...

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/37240/commits/a8f06404638e4898f0962b431c7e75a65bd21ed9#diff-8c5d3577c2bade1bdf78db032b2962213c184297ddb80606278ab85dd3264fbdR11